src/MinterGateway.sol

```diff
@@ -410,16 +410,12 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
     function deactivateMinter(address minter_) external onlyActiveMinter(minter_) returns (uint240 inactiveOwedM_) {
         if (isMinterApproved(minter_)) revert StillApprovedMinter();
 
-        uint112 principalOfActiveOwedM_ = principalOfActiveOwedMOf(minter_);
+        _imposePenaltyIfMissedCollateralUpdates(minter_);
 
         unchecked {
-            // As an edge case precaution, if the resulting principal plus penalties is greater than the max uint112,
-            // then max out the principal.
-            uint112 newPrincipalOfOwedM_ = UIntMath.bound112(
```

Comment:
NOTE: this edge case is already handled in `_imposePenalty`: https://github.com/MZero-Labs/protocol/blob/28611284d5aabd17e300249d343d6ff2c3f8ecc8/src/MinterGateway.sol?plain=1#L745
So it's safe to remove it here.

Reply:
Is this just a comment or is this suggesting a change?